emsdk.py

```diff
@@ -1383,7 +1383,7 @@ def emscripten_npm_install(tool, directory):
   print('Running post-install step: npm ci ...')
   try:
     subprocess.check_output(
-        [npm, 'ci', '--production', '--no-optional'],
+        [npm, 'ci', '--no-package-lock', '--omit=dev', '--omit=optional'],
```

Comment:
I guess `--no-package-lock` means we no longer get pinned versions but just the latest versions from package.json?

Maybe that isn't a big problem and `package.json` already specifies constrained-enough version?